### PR TITLE
added tile component

### PR DESCRIPTION
**Pull request by gbataglini**
Created a reusable component to display all destinations

### CODE REVIEW

**Comment by tigh-latte** [comment]
this files could do with prettier being ran on them

File: src/routes/upcoming.jsx [hunk 1]
```diff
@@ -15,18 +16,13 @@ export default function Upcoming() {
...
-
-
   useEffect(() => {
     fetchDestinationData();
   }, []);
 
+  useEffect(() => {}, [destinations])
+
+
   const handleClick = async () => {
```
Comment (nmcclelland) [inline src/routes/upcoming.jsx:26]: what dis do?

File: src/routes/upcoming.jsx [hunk 2]
```diff
@@ -40,6 +36,18 @@ const handleDelete = (id) => {
       console.log(err.message);
     }
   };
+  
+  const handleDelete = (id) => {
```
Comment (nmcclelland) [inline src/routes/upcoming.jsx:40]: Usually best practice to declare functions like this: 

function handleDelete(id) {
...
}

Helps with debugging.

File: src/routes/upcoming.jsx (hunk 1 above)
Comment (nmcclelland) [reply src/routes/upcoming.jsx:26]: ^ this went to the wrong place. Meant the handle click function

**Comment by nmcclelland** [comment]
> these files could do with prettier being ran on them

^^^^

File: src/routes/upcoming.jsx (hunk 1 above)
Comment (gbataglini) [reply src/routes/upcoming.jsx:30]: Randomly picks a destination, changed it to handleRandomDest

**Comment by gbataglini** [comment]
@nmcclelland ready for re-review

File: src/api/upcoming/upcomingActions.jsx [hunk 3]
```diff
@@ -0,0 +1,78 @@
+export function upcomingActions() {
```
Comment (tigh-latte) [inline src/api/upcoming/upcomingActions.jsx:1]: This could do with being renamed to something less page specific

File: src/routes/home.jsx [hunk 4]
```diff
@@ -1,31 +1,22 @@
...
-      if (res.status === 200) {
-        setDestination("");
-        console.log("Destination added successfully");
-      } else {
-        alert("Error! Could not add destination. ");
-      }
-    } catch (err) {
-      console.log(err);
+    let success = await createDestination(destination);
+    console.log(success);
```
Comment (tigh-latte) [inline src/routes/home.jsx:14]: Could get rid of the debug log

**Review by tigh-latte — APPROVED** [review]
Looking good! Couple of changes nothing blocking

**Pull request merged**